internals/daemon/api_notices.go

```diff
@@ -92,12 +128,50 @@ func v1GetNotices(c *Command, r *http.Request, _ *UserState) Response {
 		notices = st.Notices(filter)
 	}
 
-	if notices == nil {
-		notices = []*state.Notice{} // avoid null result
-	}
```

Comment:
Is there a reason to remove this? If `Notices` returns a nil slice, we want the JSON result to be `[]` rather than `null`.

Reply:
I thought the state always returned an empty slice instead of `nil`, but you're right, `flattenNotices()` can return an unset `var notices []*Notice`. I'll add it back!